Shared/MessageDialogService.cs

```diff
@@ -123,7 +132,11 @@ TResult closeButtonResult
 				while (true);
 			}
 
+#if IS_WINUI
+			var information = await DispatcherQueueExtensions.TryEnqueueAsync(_dispatcher, CreateDialogUI, DispatcherQueuePriority.Normal);
```

Comment:
```suggestion
			var information = await _dispatcher.TryEnqueueAsync(CreateDialogUI, DispatcherQueuePriority.Normal);
```

Reply:
I think it's an extension method, so you could use it like this instead :)

Reply:
In that case, it's telling me that `DispactherQueue ` (`_dispatcher`) doesn't contain `TryEnqueueAsync(...)`